perennialAutodidact/memory-snap
# install react hook form

﻿## Description of changes 
I ran yarn add react-hook-form in the frontend package folder. React hook form 7.51.1 now appears as a dependency in frontend package.json. 


## Test steps

import some basic code from the get started page into app.js and see that global functions are in scope. 

import { useForm } from "react-hook-form"

 const {
    register,
    handleSubmit,
    watch,
    formState: { errors },
  } = useForm()


https://react-hook-form.com/get-started#Quickstart
